File: src/Squidex/app/features/administration/pages/messages.ts
/*
 * Squidex Headless CMS
 *
 * @license
 * Copyright (c) Squidex UG (haftungsbeschränkt). All rights reserved.
 */

import { UserDto } from 'shared';

export class UserCreated {
    constructor(
        public readonly user: UserDto
    ) {
    }
}

export class UserUpdated {
    constructor(
        public readonly user: UserDto
    ) {
    }
}

File: backend/src/Squidex.Domain.Apps.Entities/Backup/Messages.cs
// ==========================================================================
//  Squidex Headless CMS
// ==========================================================================
//  Copyright (c) Squidex UG (haftungsbeschraenkt)
//  All rights reserved. Licensed under the MIT license.
// ==========================================================================

using Squidex.Infrastructure;

#pragma warning disable MA0048 // File name must match type name
#pragma warning disable SA1313 // Parameter names should begin with lower-case letter

namespace Squidex.Domain.Apps.Entities.Backup;

public sealed record BackupRestore(RefToken Actor, Uri Url, string? NewAppName = null);

public sealed record BackupStart(DomainId AppId, RefToken Actor);

public sealed record BackupDelete(DomainId AppId, DomainId Id);

public sealed record BackupClear(DomainId AppId);
